sdk/src/test/java/co/omise/android/ui/AuthorizingPaymentViewModelTest.kt

```diff
@@ -60,6 +60,7 @@ class AuthorizingPaymentViewModelTest {
             """-----BEGIN CERTIFICATE-----\r\nMIIDbzCCAlegAwIBAgIJANp1aztd\r\n-----END CERTIFICATE-----""",
             "TestId",
             "N2OZnlR1fDizs2SO5ukLnmaCupBaHcA=",
+            "2.2.0",
```

Comment:
Will we test with other versions?

Reply:
its just a demo number. It does not matter what we put here.